### DISCUSSION TITLE
How to update from 2016 version to 2020 version?

### DISCUSSION THREAD

**Discussion by creamyshot** [datenstrom/community#490] [Ask a question] [answered]
Hello,
I have a site running with Yellow since 2016 (never updated it) and, as I am going to work on it again, I would like to update it  - and to keep the content.
I looked for guide or whatever about "update" "how to update", but I could not find it out with the research here. The one I found was mentioning auto-update - but there is no such thing in this version... 
Could you please let me know the post that explains how to update ?
Thank you in advance for your help and patience.
Sylvain

**Comment by schulle4u** [comment, marked as answer]
The update feature only is available through the command line in this old version. However significant parts of the Yellow infrastructure have changed, therefore you probably will run into problems. I suggest to start with a fresh copy of Datenstrom Yellow. Install and configure [all needed extensions](https://github.com/datenstrom/yellow-extensions), and copy your `/content/` and `/media/` folders over the new installation. Depending on your pages, you might have to adjust some page settings, most important being the `Template` setting wich is now named `Layout`. 

Good luck!
Steffen
